### ISSUE TITLE
FR: Search results alternative

### ISSUE THREAD

**Issue by gerroon** [zadam/trilium#756]
Hi

Would not it better if we do not have a separate results window but instead , the tree would be filtered (and expanded  based on the resulting notes) depending on the search results? That way we can maximize the pane and access the results easily. At the moment the results pane cripples the tree and also cripples the itself by needing to use the same space area as the tree. We loose space from the both spaces. And it  gets very tedious to check the results when is alot of results.

**Comment by zadam** [comment]
Prior to 0.31 IIRC the tree was filtered so it worked the way you describe, but it had some usability issues.

I have plans to revisit the search results issue relatively soon though.

**Comment by nil0x42** [comment]
That would be perfect for my use case. I was about to post the same issue last week

**Comment by gerroon** [comment]
Also how about providing search results as a separate tab? That way we get to have the whole page for search results?

**Comment by zadam** [comment]
Yes, that is one possibility. However that has the issue that quickly going over the search results is quite annoying - currently I can just click on one result and I see the text in the center/right pane - if it's not it, I can click on the next and so on. If the results would be in the tab, then to see the actual notes I would have to open them in new tabs and switch between them which also isn't ideal ...

One possibility would be to have the results by default in left pane with a button to open them in a new tab ...

**Comment by gerroon** [comment]
I think offering the new tab option would be nice. The issue is that the search pane is so small and the text is wrapped around which makes grasping the results very hard especially if one has to go through hundreds of search hits.

**Comment by lerone-b** [comment]
in relation to useful search improvements – and also with a view to #1041 in process – I´d like to add that a way to let the note title stand out more clearly would be appreciated.
I do indeed like to see the whole path of a note for context – but as I am really searching for notes it is a little tough discerning that as only the _final_ part of a rather long / complex string. :-)

thx for considering!

**Comment by eliandoran** [comment]
We already have the quick search and full search, adding a tree filtering at this point would already fragment the search function too much. The full search already displays its results in the tree as sub-children.
